post_processing_stages/annotate_cv_stage.cpp

```diff
@@ -69,7 +69,7 @@ void AnnotateCvStage::Configure()
 	stream_ = app_->GetMainStream();
 	if (!stream_ || stream_->configuration().pixelFormat != libcamera::formats::YUV420)
 		throw std::runtime_error("AnnotateCvStage: only YUV420 format supported");
-	app_->StreamDimensions(stream_, &width_, &height_, &stride_);
+	app_->GetStreamInfo(stream_, &width_, &height_, &stride_);
```

Comment:
I thought we'd replaced this one by GetStreamInfo(stream_, &info) now? (Maybe I misunderstood something?)